src/helpers/scaffoldProject.ts

```diff
@@ -26,30 +27,64 @@ export const scaffoldProject = async ({
   const spinner = ora(`Scaffolding in: ${projectDir}...\n`).start();
 
   if (fs.existsSync(projectDir)) {
-    if (fs.readdirSync(projectDir).length === 0) {
+    const userFilesSet = new Set(getAllFiles(projectDir));
+    const srcFiles = [...getAllFiles(srcDir), ".gitignore"]; // template files
+    const conflictingFiles = srcFiles.filter((element) =>
```

Comment:
userFilesSet might have e.g. `prisma/schema.prisma` which would not be in srcFiles and therefore not in conflictingFiles. Continuing to scaffold and letting the user pick prisma would override the existing file which is not what we want.